### ISSUE TITLE
Regression when bundling this library

### ISSUE THREAD

**Issue by AviVahl** [microsoft/vscode-nls#44]
v5.1.0 had a regression when bundling using webpack:
```
 WARNING in ./node_modules/vscode-nls/lib/browser/main.js 26:14-44
  Module not found: Error: Can't resolve 'vscode-nls-web-data' in '/<project-path>/node_modules/vscode-nls/lib/browser'
```

regressed by #42

**Comment by TylerLeonhardt** [comment]
Just published 5.2.0 that has removed this feature.

**Comment by AviVahl** [comment]
Thanks! :+1:
